packages/engine/Source/Scene/BoundingVolumeSemantics.js

```diff
@@ -98,10 +138,18 @@ function parseBoundingVolume(prefix, tileMetadata) {
  * @return {number} The minimum height
  * @private
  */
-function parseMinimumHeight(prefix, tileMetadata) {
+BoundingVolumeSemantics.parseMinimumHeight = function (prefix, tileMetadata) {
```

Comment:
This method is not used outside the class. Can we prefix the name with an underscore to make the intended scope more clear?